index.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var path = require('path');
+var Funnel = require('broccoli-funnel');
+var mergeTrees = require('broccoli-merge-trees');
+
+module.exports = {
+  name: 'moment',
+
+  included: function(app) {
+    this._super.included(app);
+    app.import(path.join(this.treePaths.vendor, 'moment', 'min', 'moment.min.js'));
+  },
+
+  treeForVendor: function(vendorTree) {
+    var trees = [];
+    var momentPath = path.dirname(require.resolve('moment'));
+
+    if (vendorTree) {
+      trees.push(vendorTree);
+    }
+    if (momentPath) {
+      trees.push(new Funnel(this.treeGenerator(momentPath), {
```

Comment:
decided to pick all the moment files in just in case someone want locale support, those locale modules are included but not imported into the app.  i'll add maybe some configuration around this